install_config/upgrading/blue_green_deployments.adoc

```diff
@@ -0,0 +1,157 @@
+[[upgrading-blue-green-deployments]]
+= Blue-Green Deployments
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+Blue-green deployments are a proven approach to reducing downtime caused while
+updating an environment. This is done by creating a parallel environment on
+which the new deployment can be installed. If a problem is detected and after
+the new deployment is verified, traffic can be switched over with the option to
+rollback.
+
+While blue-green is a valid strategy for deploying just about any software,
+there are always trade-offs to be made. Not all environments have the same
+uptime requirements or the resources to properly perform blue-green deployments.
+In an {product-title} environment the most suitable candidate for blue-green
+deployments are the Nodes. Uptime is most important for these workloads and the
+additional complexity of blue-green deployments can be justified. Often the main
+challenge is having the excess capacity to facilitate such an approach.
+
+ifdef::openshift-enterprise[]
+Another lesser challenge is that the administrator will have to temporarily
+share the entitlements between the blue-green deployments or provide access to
+the installation content by means of a system such as Red Hat Satellite. This
+can be accomplished by sharing the consumer ID from the previous host.
+
+On the old host:
+----
+# subscription-manager identity | grep system
+system identity: 6699375b-06db-48c4-941e-689efd6ce3aa
+----
+
+On the new host:
+----
+# subscription-manager register --consumerid=6699375b-06db-48c4-941e-689efd6ce3aa
+----
+
+[IMPORTANT]
+====
+After a successful deployment, remember to unregister the old host with
+`*subscription-manager clean*` to prevent the environment from being out of
+compliance.
+====
+
+endif::openshift-enterprise[]
+
+[[manual-upgrades-preparing-for-upgrade]]
```

Comment:
Probably update these unique IDs that start with `[[manual-upgrades` (there are more below) to start with `[[blue-green` or something.